server/clip_server/model/mclip_model.py

```diff
@@ -74,4 +74,4 @@ def encode_text(
         )
 
     def encode_image(self, pixel_values: torch.Tensor, **kwargs):
-        return self._model.encode_image(pixel_values)
+        return self._model.encode_image(pixel_values, **kwargs)
```

Comment:
no, we are sure here `encode_image` only accepts `pixel_values`. Thus, `kwargs` should not be considered.